generic issue template

plamarque opened the pull request:
a generic issue template that can work for any type of issue : bugs, feature requests, feedbacks, ideas, ...

 instructions and samples include din comments

<!-- Ensure to provide github issue and task id in the title -->
<!-- Choose between feat and fix in the title to differenciate a new feature from a fix -->
<!-- Title format must be :
feat: FEATURE TITLE - MEED-XXXX - Meeds-io/MIPs#1234
or
fix: Fix TITLE - MEED-XXXX - Meeds-io/meeds#1234
-->

<!-- Description : describe the feature/the fix by answering theses questions : -->
<!-- Why is this change needed?-->
<!-- Prior to this change, ...-->
<!-- How does it address the issue?-->
<!-- This change ...-->


<!-- Tips : 
Try To Limit Each Line to a Maximum Of 72 Characters
Provide links or keys to any relevant tickets, articles or other resources

Remember to
- Capitalize the subject line
- Use the imperative mood in the subject line
- Do not end the subject line with a period
- Separate subject from body with a blank line
- Use the body to explain what and why vs. how
- Can use multiple lines with "-" for bullet points in body
-->

plamarque committed:
generic issue template

a generic issue template that can work for any type of issue : bugs, feature requests, feedbacks, ideas, ...

 instructions and samples include din comments

plamarque committed:
added config to allow blank issue template

srenault-meeds reviewed (changes requested):
> - [ ] Feedback — An observation or need regarding product performance.
- [ ] Feature Request — A proposal for new or improved functionality.
- [ ] Improvement — An enhancement to an existing feature.
- [ ] Idea — A creative or innovative suggestion.

Too much choice I think. Can we merge items please?
Suggestions:
- Feedback (merging feedback & idea)
- Feature request (merging feature request & improvement)

Plus, the order would be easier when writing as is:
1. Bug
2. Feature Request
3. Feedback

> Current behaviour part

Redondant with "Observe the issue". Most of the times, I have observed that user duplicates answers here. 
IMO observe the issue is equal to current behaviour
So we remove current behaviour part

> Acceptance criteria

To suggest only for bug report

> Expected benefits

To suggest only for feature request

plamarque committed:
Merge branch 'develop' into plamarque-patch-1

plamarque committed:
Reduced the options

- merged Feedback and Idea
- merged Feature request and Improvement

plamarque committed:
Optional secitons

- renamed feature requests to Feature for simplivity
- marked acceptance criteria as required for bugs only and expected benefits for features only

plamarque commented:
@srenault-meeds I applied your suggested changeS. There is no easy way to have a conditional form. I'm looking for more advanced form capabilities in GitHub, but it will be a separate tempalte. I think we can start with this one and see if we can build a smarter conditional form separately

srenault-meeds reviewed (approved):
LGTM! Thanks
